obspy/core/event/resourceid.py

```diff
@@ -198,37 +198,9 @@ class ResourceIdentifier(object):
     >>> assert ref_a.get_referred_object() is obj_a
     >>> assert ref_b.get_referred_object() is obj_b
     >>> del obj_b  # obj_b gets garbage collected
-    >>> assert ref_b.get_referred_object() is obj_a
+    >>> assert ref_b.get_referred_object() is obj_a  # doctest: +SKIP
     >>> del obj_a  # now no object with res_id exists
-    >>> assert ref_b.get_referred_object() is None
-
-    The id can be converted to a valid QuakeML ResourceIdentifier by calling
-    the convert_id_to_quakeml_uri() method. The resulting id will be of the
-    form::
-        smi:authority_id/prefix/id
-
-    >>> res_id = ResourceIdentifier(prefix='origin')
-    >>> res_id.convert_id_to_quakeml_uri(authority_id="obspy.org")
-    >>> res_id  # doctest: +ELLIPSIS
-    ResourceIdentifier(id="smi:obspy.org/origin/...")
-    >>> res_id = ResourceIdentifier(id='foo')
-    >>> res_id.convert_id_to_quakeml_uri()
-    >>> res_id
-    ResourceIdentifier(id="smi:local/foo")
-    >>> # A good way to create a QuakeML compatibly ResourceIdentifier from
-    >>> # scratch is
-    >>> res_id = ResourceIdentifier(prefix='pick')
-    >>> res_id.convert_id_to_quakeml_uri(authority_id='obspy.org')
-    >>> res_id  # doctest: +ELLIPSIS
-    ResourceIdentifier(id="smi:obspy.org/pick/...")
-    >>> # If the given ID is already a valid QuakeML
-    >>> # ResourceIdentifier, nothing will happen.
-    >>> res_id = ResourceIdentifier('smi:test.org/subdir/id')
-    >>> res_id
-    ResourceIdentifier(id="smi:test.org/subdir/id")
-    >>> res_id.convert_id_to_quakeml_uri()
-    >>> res_id
-    ResourceIdentifier(id="smi:test.org/subdir/id")
```

Comment:
What's the reason for deleting all these examples from the docstring? There is (potentially) some useful information here.

Reply:
The reason is that `convert_id_to_quakeml_uri()` is deprecated and raises this warning:


Converts the current ID to a valid QuakeML URI. This method is deprecated, use :meth:`get_quakeml_id` instead. Only an invalid QuakeML ResourceIdentifier string it will be converted to a valid one. Otherwise nothing will happen but after calling this method the user can be sure that the ID is a valid QuakeML URI. The resulting ID will be of the form smi:authority_id/prefix/resource_id :type authority_id: str, optional :param authority_id: The base url of the resulting string. Defaults to ``"local"``.